flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -56,7 +57,7 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
     public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
         switch call.method as String {
         case "loadContexts":
-            loadContexts(result: result)
+            loadContexts(call, result: result)
```

Comment:
On Android we have a specific method call for loading image list, called `loadImageList`, can we reuse the same integration instead of returning it along with the `loadContexts`?

Reply:
changed